spec/controllers/chargeback_controller_spec.rb

```diff
@@ -26,7 +26,10 @@
       before do
         cbr = FactoryGirl.create(:chargeback_rate, :rate_type => "Storage")
         ChargebackRate.set_assignments(:Storage, [{:cb_rate => cbr, :tag => [tag, "vm"]}])
-        sandbox = {:active_tree => :cb_assignments_tree, :trees => {:cb_assignments_tree => {:active_node => 'xx-Storage'}}}
+        sandbox = {
+          :active_tree => :cb_assignments_tree,
+          :trees => {:cb_assignments_tree => {:active_node => 'xx-Storage'}}
+        }
```

Comment:
I was wrong @aljesusg, the link in the email brought me to the old stale commit view.  I see the change now.  Please align the two `=>` vertically above :point_up: